SimG4Core/Geometry/interface/DDDWorld.h

```diff
@@ -4,19 +4,22 @@
 #include "SimG4Core/Geometry/interface/G4LogicalVolumeToDDLogicalPartMap.h"
 #include "SimG4Core/Geometry/interface/SensitiveDetectorCatalog.h"
 #include "G4VPhysicalVolume.hh"
-#include "DDG4/Geant4GeometryInfo.h"
 
-class DDG4Builder;
 class DDCompactView;
 
 namespace cms {
-  class DDDetector;
-}  // namespace cms
+  class DDCompactView;
+}
 
 class DDDWorld {
 public:
+  DDDWorld(const DDCompactView *pDD,
+           const cms::DDCompactView *pDD4hep,
```

Comment:
@civanch so for the time being you are always assuming both geometries in the interface, just one of the two will be a ```nullptr``` , but the interface is now universal and unique, right?

Reply:
@fabiocos , yes, exactly.